Annotations/Route.php

```diff
@@ -37,16 +38,41 @@
     private $route;
 
     /**
+     * @var string
+     */
+    private $method;
+
+    /**
+     * @var int
+     */
+    private $statusCode;
+
+    /**
+     * @var array
+     */
+    private $options = [];
+
+    /**
      * Constructor
      *
      * @param array $values
      *
      * @SuppressWarnings("PHPMD.StaticAccess")
      */
     public function __construct(array $values) {
-        $route = $values['value'];
-        if (!Assertions::isUrl($route)) return Exceptions::InvalidTypeException('Url', 'route', $route);
-        $this->route = $route;
+        $settings = new ArrayCollection($values);
+
+        $this->route = $settings->get('value');
+        Assertions::assertUrl('value', $this->route);
```

Comment:
wat soll das hier? wieso nich einfach:

```php
$this->route = Assertions::assertUrl($settings->get("value"));
...
```

(falls das der selector is: der hat in ner assertion in meinen augen absolut nix verloren und vor allem nich als erstes argument)